spec/turnip_steps.rb

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
Could you move this file into  support folder. `support/turnip_step.rb`

Reply:
Is `turnip_step.rb` prefer being in `support`  to being in `support/steps` ?
I think that `support/steps` is more suitable.

Reply:
yeah, steps directory is better.